sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -478,6 +479,22 @@ def process(
       self.assertEqual(1, len(counters))
       self.assertEqual(counters[0].committed, len(''.join(data)))
 
+  def test_sdf_wrap_range_source(self):
+    with self.create_pipeline() as p:
+      from apache_beam.options.pipeline_options import DebugOptions
+      experiments = (p._options.view_as(DebugOptions).experiments or [])
+
+      # Setup experiment option to enable using SDFBoundedSourceWrapper
+      if not 'use_sdf_bounded_source' in experiments:
+        experiments.append('use_sdf_bounded_source')
+
+      p._options.view_as(DebugOptions).experiments = experiments
+
+      actual = (
+          p | beam.io.Read(RangeSource(0, 4))
+      )
+      assert_that(actual, equal_to([0, 1, 2, 3]))
```

Comment:
This won't verify that the PTransform replacement actually occurred since if the replacement didn't occur, you would still get the same pipeline result.

Reply:
Adding one more test case to verify overrides: https://github.com/apache/beam/pull/8641/commits/2392ae94101c71135466837547838d4b8bc1e090